### PR TITLE
Improve dutch translation

### PR DESCRIPTION
**Pull request by Ka-Wing**
See [JENKINS-49901](https://issues.jenkins-ci.org/browse/JENKINS-49901).

Issue should say enough. As these are text translations, I don't think tests are needed.
I am a native speaker by the way.

### Proposed changelog entries

* Improved Dutch translation.

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

Dutch natives.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
CC @vStone who may be interested (https://github.com/jenkinsci/jenkins/pull/3286)

**Comment by oleg-nenashev** [comment]
@lange88 are you fine with the changes?

File: core/src/main/resources/hudson/util/IncompatibleServletVersionDetected/index_nl.properties [hunk 1]
```diff
@@ -22,5 +22,5 @@
 
 Error=Fout
 errorMessage=\
-  Blijkbaar ondersteunt uw servlet container de servlet versie 2.4 niet!\
+  Blijkbaar ondersteunt uw servletcontainer de servlet versie 2.4 niet!\
```
Comment (vStone) [inline core/src/main/resources/hudson/util/IncompatibleServletVersionDetected/index_nl.properties]: `Blijkbaar` bugs me here. `We merken dat` ?

File: core/src/main/resources/hudson/util/NoTempDir/index_nl.properties [hunk 2]
```diff
@@ -25,6 +25,6 @@ description=\
  Kon geen tijdelijk bestand creëeren!  De meest waarschijnlijke oorzaak\
 hiervoor is een verkeerde configuratie van de container.\
 De JVM werd geconfigureerd om volgende folder als tijdelijke \
-folder te gebruiken :{0}\
-Bestaat deze folder?  Is deze folder beschrijfbaar door de gebruiker\
+map te gebruiken :{0}\
+Bestaat deze map?  Is deze map beschrijfbaar door de gebruiker\
```
Comment (vStone) [inline core/src/main/resources/hudson/util/NoTempDir/index_nl.properties]: `beschrijfbaar` -> `wijzigbaar`

File: core/src/main/resources/jenkins/model/Jenkins/projectRelationship-help_nl.properties [hunk 3]
```diff
@@ -22,11 +22,11 @@
...
 body=\
-  Wanneer je projecten ontwikkelt, die van elkaar afhankelijk zijn, kan Jenkins voor jou uitzoeken welke \
-  bouwpoging van een bovenliggend project, gebruikt wordt door een onderliggend project. Dit gebeurt aan \
+  Wanneer je projecten ontwikkelt die van elkaar afhankelijk zijn, kan Jenkins voor jou uitzoeken welke \
+  bouwpoging van een bovenliggend project gebruikt wordt door een onderliggend project. Dit gebeurt aan \
   de hand van de geregistreerd <a href="https://jenkins.io/redirect/fingerprint">elektronische vingerafdrukken</a> van \
-  de,door een bouwpoging, opgeleverde artefacten.
-For\ this\ feature\ to\ work,\ the\ following\ conditions\ need\ to\ be\ met\:=Aan volgende voorwaarden dient voldaan om met deze functionaliteit te werken:
+  de door een bouwpoging opgeleverde artefacten.
+For\ this\ feature\ to\ work,\ the\ following\ conditions\ need\ to\ be\ met\:=De volgende voorwaarden dient voldaan te worden om met deze functionaliteit te werken:
```
Comment (vStone) [inline core/src/main/resources/jenkins/model/Jenkins/projectRelationship-help_nl.properties]: `De volgende voorwaarden dienen voldaan...`

File: core/src/main/resources/lib/hudson/project/config-publishers_nl.properties [hunk 4]
```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Post-build\ Actions=Acties na bouwpoging
\ No newline at end of file
+Post-build\ Actions=Acties voor na de bouwpoging
```
Comment (vStone) [inline core/src/main/resources/lib/hudson/project/config-publishers_nl.properties]: `Acties uit te voeren na de bouwpoging`

File: core/src/main/resources/lib/hudson/project/config-publishers2_nl.properties [hunk 5]
```diff
@@ -20,5 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Add\ post-build\ action=Toevoegen actie na bouwpoging
-Post-build\ Actions=Acties na bouwpoging
+Add\ post-build\ action=Toevoegen actie voor na de bouwpoging
+Post-build\ Actions=Acties voor na de bouwpoging
```
Comment (vStone) [inline core/src/main/resources/lib/hudson/project/config-publishers2_nl.properties]: `Acties uit te voeren` na / voor `de bouwpoging`

File: core/src/main/resources/lib/layout/breadcrumbBar_nl.properties [hunk 6]
```diff
@@ -1,5 +1,5 @@
 # This file is under the MIT License by authors
 
-DISABLE\ AUTO\ REFRESH=ZET AUTO VERVERSING UIT
+DISABLE\ AUTO\ REFRESH=ZET AUTOMATISCH VERNIEUWEN UIT
 
-ENABLE\ AUTO\ REFRESH=Automatisch herladen inschakelen
+ENABLE\ AUTO\ REFRESH=Automatisch vernieuwen inschakelen
```
Comment (vStone) [inline core/src/main/resources/lib/layout/breadcrumbBar_nl.properties]: Capitalization should match original string

**Review by vStone — CHANGES_REQUESTED** [review]
I've went through it, finally. some remarks added.

File: core/src/main/resources/hudson/model/Fingerprint/index_nl.properties [hunk 7]
```diff
@@ -22,5 +22,5 @@
 
 introduced={0} geleden ge\u00EFntroduceerd
 outside\ Jenkins=buiten Jenkins
-This\ file\ has\ not\ been\ used\ anywhere\ else.=Dit bestand werd nog nergens anders gebruikt.
-This\ file\ has\ been\ used\ in\ the\ following\ places=Dit bestand werd op de volgende plaatsen gebruikt.
+This\ file\ has\ not\ been\ used\ anywhere\ else.=Dit bestand is nergens anders gebruikt.
+This\ file\ has\ been\ used\ in\ the\ following\ places=Dit bestand is volgende plaatsen gebruikt.
```
[the remark this answers is not in the document]
Comment (Ka-Wing) [reply core/src/main/resources/hudson/model/Fingerprint/index_nl.properties]: Yeah, should be "Dit bestand is op de volgende plaatsen gebruikt.".

File: core/src/main/resources/hudson/model/Run/logKeep_nl.properties [hunk 8]
```diff
@@ -20,5 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Don't\ keep\ this\ build\ forever=Deze bouwpoging niet eeuwig bijhouden.
-Keep\ this\ build\ forever=Bewaar deze Build eeuwig
+Don't\ keep\ this\ build\ forever=Deze bouwpoging niet voor altijd bijhouden.
+Keep\ this\ build\ forever=Bewaar deze Build voor altijd.
```
[the remark this answers is not in the document]
Comment (Ka-Wing) [reply core/src/main/resources/hudson/model/Run/logKeep_nl.properties]: Indeed, I have overlooked it. I have changed it to "bouwpoging"

**Comment by Ka-Wing** [comment]
@vStone I have made changes according to your suggestions. Can you check them again please?

**Comment by lange88** [comment]
Changes look good @Ka-Wing @oleg-nenashev

**Review by vStone — APPROVED** [review]
👍

**Comment by oleg-nenashev** [comment]
Thanks a lot @Ka-Wing @lange88 @vStone !

**Comment by Ka-Wing** [comment]
@oleg-nenashev You are welcome. I am happy to be able to help.

**Pull request merged**